cmd/webhook-manager/app/server.go

```diff
@@ -79,7 +79,7 @@ func Run(config *options.Config) error {
 	})
 
 	webhookServeError := make(chan struct{})
-	stopChannel := make(chan os.Signal)
+	stopChannel := make(chan os.Signal, 1)
```

Comment:
what's the reason for using a buffered channel?

Reply:
With the staticcheck tool I get a warning message that reads 
`cmd/webhook-manager/app/server.go:83:16: the channel used with signal.Notify should be buffered (SA1017)`

I think the stopChannel is a goroutine designed to notify messages, and if it is unbuffered, then sending messages to it, or receiving them, will be blocked.